pkg/k8sutil/cluster.go

```diff
@@ -1,70 +1,57 @@
 package k8sutil
 
 import (
+	"context"
+	"fmt"
+	v1 "k8s.io/api/core/v1"
```

Comment:
move this into the section below the space, and rename the import to `corev1` for consistency